index.html

```diff
@@ -139,6 +140,19 @@ <h2>Methods</h2>
         </dd>
       </dl>
     </section>
+    <section>
+      <h2>Modifications to existing procedures</h2>
+
+      <p>Whenever the <a data-cite="WEBRTC#dom-peerconnection">RTCPeerConnection constructor</a>
+      algorithm is invoked, run the following steps instead:
+        <ol>
+          <li>If [[WEBAPPSEC-CSP#should-block-rtc-connection]] on the current global object returns "Blocked",
+          throw a {{NotAllowedError}}."</li>
+          <li>Return the result of running the original
+          <a data-cite="WEBRTC#dom-peerconnection">RTCPeerConnection constructor</a> algorithm.</li>
```

Comment:
The constructor algorithm has its own tag: https://w3c.github.io/webrtc-pc/#constructor

It's probably to use that rather than linking to the whole peerconnection definition.